.travis.yml

```diff
@@ -19,7 +19,7 @@ before_install:
   - psql -c "alter user postgres with password 'Password12!';" -U postgres
 
 env:
-  - CXX=g++-4.8
+  - CXX=g++-4.8 GCLOUD_TRACE_NEW_CONTEXT=1
```

Comment:
Separate lines means disjoint run configurations (one set of tests at each node version with `CXX=g++-4.8` and another set of tests at each node version with `GCLOUD_TRACE_NEW_CONTEXT=1`. Writing them on the same line applies both env vars to each test configuration. https://docs.travis-ci.com/user/environment-variables/#Defining-Multiple-Variables-per-Item